xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/po/EditRightsPane.java

```diff
@@ -71,7 +73,9 @@ State getNextState()
         static State getButtonState(WebElement button)
         {
             for (State s : values()) {
-                if ((button.getAttribute("src").endsWith(s.imageURL))) {
+                // starting with 11.1RC1 the resource URLs can now contain a query parameter to avoid cache issue
+                // so we cannot rely to a full URL anymore
+                if ((button.getAttribute("src").contains(s.imageURL))) {
```

Comment:
I would put a more generic message, that we use contains() because the URL may contain query string parameters (e.g.  starting with 11.1RC1 the resource URLs can now contain a query parameter to avoid cache issue) and we don't care about that to identify the state.

Reply:
OK I'll improve it.